schema.sql

```diff
@@ -0,0 +1,9 @@
+/* Database schema to keep the structure of entire database. */
+
+create table animals(id INT PRIMARY KEY NOT NULL, 
+					 name TEXT NOT NULL, 
+					 date_of_birth DATE NOT NULL, 
+					 escape_attempts INT DEFAULT 0, 
+					 neutered BOOLEAN DEFAULT false, 
+					 weight_kg REAL
+					);
```

Comment:
- [ ] I suggest you change the data type for `name` to `varchar(255)` or varchar with any number of string lengths you wish to have. It is a requirement to have `varchar`  as the data type for the `name`.

- [ ] Also kindly change the data type for `weight_kg` to `DECIMAL` instead of `REAL` as this is the requirement. 

```suggestion
					 name VARCHAR(255) NOT NULL, 
					 date_of_birth DATE NOT NULL, 
					 escape_attempts INT DEFAULT 0, 
					 neutered BOOLEAN DEFAULT false, 
					 weight_kg DECIMAL
					);
```

Reply:
Done, thank you.